WordPress/Classes/Services/ReaderPostService.m

```diff
@@ -441,6 +441,18 @@ - (void)toggleSeenForPost:(ReaderPost *)post
                   success:(void (^)(void))success
                   failure:(void (^)(NSError *error))failure;
 {
+    if (post.feedItemID == nil) {
```

Comment:
Could be a `guard` but meh. I'm not a stickler about such things ;)

Reply:
Oh only if ObjC had `guard`...

Reply:
🤦